.changeset/twenty-pants-fetch.md

```diff
@@ -0,0 +1,11 @@
+---
+'@sveltejs/package': minor
+'@sveltejs/kit': minor
+---
+
+feat: support svelte.config.ts
+
+> **NOTE**
+>
+> Your runtime has to support importing TypeScript files for svelte.config.ts to work.
+> In nodejs you can enable it with the `--experimental-strip-types` flag
```

Comment:
I don't think you need the flag on the latest Node 24. https://github.com/nodejs/node/pull/56350

Reply:
it doesn't break if you still pass that flag or do you think we have to be specific and list each minor where it was added as a flag and then each that unflags it?

Reply:
```suggestion
> In Node.js, the feature is supported with the `--experimental-strip-types` flag starting in Node 22.6.0 and supported without a flag starting in Node 23.6.0.
```